app/src/main/java/com/example/atheneum/fragments/BorrowerFragment.java

```diff
@@ -0,0 +1,163 @@
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.example.atheneum.fragments;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.atheneum.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BorrowerFragment extends Fragment {
+    private View view;
+    private Context context;
+    private ViewPager mPager;
+    private PagerAdapter pagerAdapter;
+
+    private  static  final int NUM_PAGES = 2;
+
+//    private OnFragmentInteractionListener mListener;
+
+    public BorrowerFragment() {
+        // Required empty public constructor
+    }
+
+
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        this.view = inflater.inflate(R.layout.fragment_borrower, container, false);
+
+        this.context = getContext();
+
+        mPager = (ViewPager) this.view.findViewById(R.id.borrowerViewPager);
+        setUpViewPager(mPager);
+
+        TabLayout tabs = (TabLayout) view.findViewById(R.id.borrowerTabLayout);
+        tabs.setupWithViewPager(mPager);
+//        pagerAdapter  = new TabsAdapter(getSupportFragmentManager(), tabLayout.getTabCount());
+
+        return this.view;
+    }
+
+    private void setUpViewPager(ViewPager viewPager){
+        TabsAdapter adapter = new TabsAdapter(getChildFragmentManager());
+        adapter.addFragment(new BorrowedBooksFragment(), "Borrowed Books");
+        adapter.addFragment(new BorrowerRequestsFragment(), "Borrower Requests");
+        mPager.setAdapter(adapter);
+
+    }
+
+//    // TODO: Rename method, update argument and hook method into UI event
```

Comment:
Are you still using this code or have intentions to use it in the future? If not, then you should just remove it.